README.md

```diff
@@ -4,9 +4,12 @@
 
 **This is not an officially supported Google product.**
 
-##Testing Library for Smart Home 
+## Testing Library for Smart Home
 
 
 The objective is to design a scalable integration testing library for Smart Home Actions. With the enhanced logging, developers will be able to find more details about failures.
 The big impact of this will be removing friction and latency from the dev cycle, which will allow the developer to successfully verify their action more efficiently.
 
+## Setup
+To run this project, install it locally using npm:
+$npm install testing-library
```

Comment:
This section can be removed, as this should not be available to install.

Reply:
Wouldn't this be something like `npm install --save-dev @google/smart-home-testing` or something so a dev can integrate it into their unit tests?

Reply:
Yes that would be part of a larger installation guide, although the package itself is not published on NPM yet so any installation instruction would not be valid at the moment.